.github/workflows/release-tagging.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Tagging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tagging:
+    runs-on: ubuntu-latest
+    env:
+      EXCLUDED_PACKAGE_OA_RENDERERS: '!./packages/vckit-oa-renderers/'
+      EXCLUDED_PACKAGE_CREDENTIAL_OA: '!./packages/credential-oa/'
+      EXCLUDED_PACKAGE_EXPLORER: '!./packages/demo-explorer/'
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install pnpm
+      run: npm i pnpm --global
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20'
```

Comment:
Should we use `node-version` 21?
<img width="171" alt="image" src="https://github.com/uncefact/project-vckit/assets/59736798/06ac5907-1dca-4414-b18d-f4e54eec143e">